packages/services/src/models/notification-scanners/AppealsOpened.js

```diff
@@ -0,0 +1,47 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+
+class AppealsOpened extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const fourDaysBeforeNow = Math.floor((Date.now()-(4*this._DAYS))/1000)
```

Comment:
@bingen you mentioned there should be a better way to get these day periods from blockchain, not sure how

Reply:
I created this [PR](https://github.com/aragonone/court-backend/pull/170) to provide that information.

You should be able to get a Court wrapper for the network you are working with and call [`config`](https://github.com/aragonone/court-backend/pull/170/files#diff-e097297aec2c8218887bb696a62fbc74R87) passing the [term ID of the dispute](https://github.com/aragon/court-subgraph/blob/master/schema.graphql#L68) we are sending the notification form (we will need to add this join to the query). 

Once you have the config, the values you're looking for are the [round durations](https://github.com/aragonone/court-backend/pull/170/files#diff-e097297aec2c8218887bb696a62fbc74R96). Remember this values are in "court terms", you should multiply them by the [term duration](https://github.com/aragonone/court-backend/pull/170/files#diff-e097297aec2c8218887bb696a62fbc74R75) (measured in seconds not milliseconds)

Reply:
as discussed, added checks with termIds